src/github-stats.ts

```diff
@@ -0,0 +1,498 @@
+// TODO github api model
```

Comment:
```suggestion
// TODO(issue#the_issue_number): define Github API entities.
```
This format is not only for Webstorm to automatically parse and highlight them in the IDE but also allowing us, as a team to track future items!